### ISSUE TITLE
Bug: [2.1.1 branch] Element.boundingBox() fails strangely for non-visible elements

### ISSUE THREAD

**Issue by bland328** [pyppeteer/pyppeteer#210]
Using the 2.1.1 branch, instead of getting `None` in the case that an element has no bounding box, I see `An exception occurred: Protocol error (DOM.getBoxModel): Could not compute box model.` in the console output.

I've tried this crude (because of the naked `except:`) workaround:

```py
    try:
        box = await el.boundingBox()
    except:
        box = None
```

However, this doesn't work, as apparently this exception is output to the console _(perhaps owing to some debugging output I've activated?)_, but doesn't actually ripple up to my code.

**Comment by Mattwmaster58** [comment]
Hmm, looks like we're catching the error and logging it ourselves (as in, the library is catching it before you). The logging setup on the branch is by no means final - chances are there's a stream handler attached by default that's doing that. In the eventual release we'll probably remove that stream handler (in fact it might be gone by now :)).

The reasoning is probably

1. That's how puppeteer does it.
2. At least logging it allows a starting point for debugging (why is this function returning None? Let's check the logs)

So basically, it's intended behavior that pyppeteer swallows this. Hope this helps. Feel free to re-open if you disagree

**Comment by Mattwmaster58** [comment]
Relevant code: https://github.com/pyppeteer/pyppeteer/blob/dcb53b760b1db09f4160aa9aa370eb7a097df5d6/pyppeteer/jshandle.py#L211-L215
